kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -198,10 +202,29 @@
     },
     $trNameSpace: 'userPage',
     $trs: {
-      noUsersExist: 'No Users Exist.',
-      allUsersFilteredOut: 'No users match the filter.',
+      // input & accessibility labels
```

Comment:
> being that verbose would gunk up the markup to the point that reading though it would get pretty frustrating

Yes, I can see that. I also find it frustrating to have the strings defined in the `<script>` section rather than inline in the `<html>` because it means scrolling up and down repeatedly. I really like the inline syntax suggested in this issue: https://github.com/learningequality/vue-intl/issues/5

> if `Admin` is used in `Learn` and `Manage`, do they have to translate it twice?

Yes. I don't have a reference offhand, I've read that since translations are often context-dependent, attempts to 'DRY' them up lead to subpar/sloppy translations.